lib/chitter.rb

```diff
@@ -0,0 +1,79 @@
+require './database_connection_setup'
```

Comment:
Having code execute on 'require' seems like an antipattern to me. Instead, this code should be in a function that you can call at the beginning of your app (just after line 11)